pages/index.html

```diff
@@ -58,31 +58,35 @@
 			</tr>
 			<tr>
 				<td class="txtright">PHP</td>
-				<td><strong>PHP 5.3.7+</strong></td>
-				<td>PHP 5.2+</td>
+				<td><strong>PHP 7</strong></td>
+				<td>older PHP versions*: 5.5+, 5.4+ or 5.3.3+</td>
 			</tr>
 			<tr>
 				<td class="txtright">PHP modules</td>
 				<td>
-					<p>Required: <a href="http://php.net/xml">libxml</a>, <a href="http://php.net/curl">cURL</a>, <a href="http://php.net/pdo-mysql">PDO_MySQL</a> or <a href="http://php.net/pdo-sqlite">PDO_SQLite</a>, <a href="http://php.net/gmp">GMP</a></p>
-					<p>Recommended: <a href="http://fr2.php.net/json">JSON</a>, <a href="http://php.net/zlib">Zlib</a>, <a href="http://php.net/mbstring">mbstring</a>, <a href="http://php.net/iconv">iconv</a>, <a href="http://php.net/zip">Zip</a></p>
+					<p>Required: <a href="http://php.net/xml">libxml</a>, <a href="http://php.net/curl">cURL</a>, <a href="http://php.net/dom">DOM</a> <a href="http://php.net/pdo-mysql">PDO_MySQL</a></p>
+					<p>Recommended: <a href="http://fr2.php.net/json">JSON</a>, <a href="http://php.net/gmp">GMP</a>, <a href="http://php.net/zlib">Zlib</a>, <a href="http://php.net/intl.idn">IDN</a>, <a href="http://php.net/mbstring">mbstring</a>, <a href="http://php.net/iconv">iconv</a>, <a href="http://php.net/zip">Zip</a>, <a href="http://php.net/zlib">zilb</a></p>
 				</td>
 				<td>
-
+                    <p>Database: <a href="http://php.net/pdo-sqlite">PDO_SQLite</a> or <a href="http://php.net/pdo-pgsql">PDO_PGSQL</a></p>
```

Comment:
MySQL is in the "Required" first column. SQLite and PostgreSQL are recommended too.